websites/B/BBC/metadata.json

```diff
@@ -26,8 +27,8 @@
 		"www.bbc.com"
 	],
 	"version": "2.2.5",
-	"logo": "https://cdn.rcd.gg/PreMiD/websites/B/BBC/assets/logo.png",
-	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/B/BBC/assets/thumbnail.png",
+	"logo": "https://i.imgur.com/EuQ5Aid.png",
```

Comment:
Not sure what changed with the logo in particular, but either way it must be 512x512.